google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/scrape/metadata_scraper.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from requests import sessions
+
+from . import authenticator, constants, repository_services_api_helper
+
+
+class MetadataScraper:
+
+    def __init__(self, server_address, domain, username, password):
+        self.__server_address = server_address
+        self.__domain = domain
+        self.__username = username
+        self.__password = password
+        self.__session = sessions.Session()
+        self.__qrs_api_helper = \
+            repository_services_api_helper.RepositoryServicesAPIHelper(
+                server_address)
+
+    def scrape_streams(self):
+        self.__set_qps_session_cookie()
+        return self.__qrs_api_helper.get_streams(self.__session)
+
+    def __set_qps_session_cookie(self):
```

Comment:
same about the `QPS` constant suggestion.

Reply:
Same reason...